requirements.txt

```diff
@@ -5,7 +5,6 @@ tqdm
 sox
```

Comment:
@okuchaiev Why do we have a requirements.txt? We have a setup.py script for each library.

Reply:
@redoctopus or @borisfom I think some container uses it, no?

Reply:
The Dockerfile refers to it: https://github.com/NVIDIA/NeMo/blob/master/Dockerfile#L34